### ISSUE TITLE
Latest 0.3.3 release does not install

### ISSUE THREAD

**Issue by mdeltito** [apollographql/persistgraphql#40]
Installing this package appears to have broken in 50970f76b3b46e4c5327895f93990d5681affede

```
mdeltito@abc $ mkdir test && cd test && npm init
mdeltito@abc $ npm install --save persistgraphql
npm WARN test@1.0.0 No description
npm WARN test@1.0.0 No repository field.
npm ERR! path /Users/mdeltito/code/test/node_modules/persistgraphql/bin/persistgraphql
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/mdeltito/code/test/node_modules/persistgraphql/bin/persistgraphql'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

**Comment by emmanuelbuah** [comment]
Yeah, thats because the commit https://github.com/apollographql/persistgraphql/commit/29cef1e908e52777f4894f543d427381d21bbef3 excluded the bin folder in the repo, hence breaking the path to the bin folder.

**Comment by Poincare** [comment]
Ah darn, will fix this right away.

**Comment by Poincare** [comment]
This should be fixed as of 0.3.5. Thanks for pointing this out.
